src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+    margin-left: 5%;
+  }
+
+  .seven-day-fc .icon {
+    width: 64px;
+    height: 64px;
+  }
+}
+```
+
+Similarly, for the large screens it's best to limit to maximum width of the forecast panel
+so it doesn't consume the whole screen width.
+
+```css
+@media (min-width: 700px) {
+  .weather-forecast {
+    width: 700px;
+  }
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/responsive-forecast?path=style.css&previewSize=100"
+    title="responsive-forecast on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Optimize text for reading
+
+Classic readability theory suggests that an ideal column should contain 70 to 80 characters per line
+(about 8 to 10 words in English).
+Thus, each time the width of a text block grows past about 10 words,
+consider adding a breakpoint.
+
+<figure class="w-figure">
+  <img src="./readability-narrow.png" width="400" alt="Screenshot of a a page of text on a mobile device">
+  <figcaption class="w-figcaption">
+    The text as read on a mobile device.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./readability-wide.png" alt="Screenshot of a a page of text on a desktop browser">
+  <figcaption class="w-figcaption">
+    The text as read on a desktop browser with a breakpoint added to constrain the line length.
+  </figcaption>
+</figure>
+
+Let's take a deeper look at the above blog post example.
+On smaller screens, the Roboto font at `1em` works perfectly giving 10 words per line,
+but larger screens require a breakpoint.
+In this case, if the browser width is greater than `575px`, the ideal content width is `550px`.
+
+```css
+@media (min-width: 575px) {
+  article {
+    width: 550px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+```
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/rwd-reading?path=index.html&previewSize=100"
+    title="rwd-reading on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Avoid simply hiding content
+
+Be careful when choosing what content to hide or show depending on screen size.
+Don't simply hide content just because you can't fit it on the screen.
+Screen size is not a definitive indication of what a user may want.
+For example, eliminating the pollen count from the weather forecast
+could be a serious issue for spring-time allergy sufferers who need the information
+to determine if they can go outside or not.
+
+## View media query breakpoints in Chrome DevTools {: #devtools }
+
+Once you've got your media query breakpoints set up,
+you'll want to see how your site looks with them.
+You could resize your browser window to trigger the breakpoints,
+but there's a better way: Chrome DevTools.
+The two screenshots below demonstrate using DevTools to view how a page looks under different breakpoints.
```

Comment:
```suggestion
Once you've got your media query breakpoints set up,
you'll want to see how your site looks with them.
You could resize your browser window to trigger the breakpoints,
but Chrome DevTools has a built-in feature that makes it easy to see how a page looks
under different breakpoints.
```
I think I originally added this text to the guide and I'm now realizing that it sound sales-y